Gemfile.lock

```diff
@@ -110,24 +110,24 @@ GEM
     crass (1.0.6)
     cssbundling-rails (1.1.1)
       railties (>= 6.0.0)
-    debug (1.6.1)
+    debug (1.6.2)
       irb (>= 1.3.6)
       reline (>= 0.3.1)
     diff-lcs (1.5.0)
     digest (3.1.0)
     docile (1.4.0)
-    dotenv (2.7.6)
-    dotenv-rails (2.7.6)
-      dotenv (= 2.7.6)
+    dotenv (2.8.1)
+    dotenv-rails (2.8.1)
+      dotenv (= 2.8.1)
       railties (>= 3.2)
-    erubi (1.10.0)
-    faraday (2.3.0)
-      faraday-net_http (~> 2.0)
+    erubi (1.11.0)
+    faraday (2.5.2)
+      faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-net_http (2.0.3)
-    gds-sso (16.1.0)
+    faraday-net_http (3.0.0)
+    gds-sso (17.0.0)
```

Comment:
We seem to be bumping more than just govuk_components/ view_components gem

Reply:
Hmm - I did have issues which meant I deleted the lock file and installed again. I'll go through them in detail.

Good catch!

Reply:
could potentially, do it as a separate pr but its up to you.